spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+RSpec.describe "UsersController" do
+include AuthenticationSpecHelper
+  before do
+    @controller = UsersController.new
+  end
+
+  describe 'GET #new' do   
+    context 'Render template new' do
```

Comment:
Bắt đầu context với when, with, without
https://www.betterspecs.org/#contexts
```suggestion
    context 'When call `new`' do
```